docs/user/collected/metrics.md

```diff
@@ -0,0 +1,63 @@
+<!-- AUTOGENERATED BY glean_parser.  DO NOT EDIT. -->
```

Comment:
I know this is all auto-generated. Should we still make it clear that these are Android-specific metrics?

Reply:
This file needs to be added to the SUMMARY.md to get rendered into the final documentation.
I'm also not sure I like the "collected" name for the directory.

I'd be okay with having this as a sub-chapter, so we can make it clear that we have some different metrics on Android on iOS instead of modifying the auto-generated file.

Reply:
Yes, it's a bit weird to have the directory called `collected`. I could not come up with a better name :( Gladly taking suggestions!

I did not link it to the main docs, because I wasn't sure how we want to handle docs, given that it might have platform differences. This directly boils down to how we want to handle the `metrics.yaml`. If we have one file shared across all the platforms (we don't expect to have many platform specific metrics, don't we?), then this will be the only file.

Do we already have iOS only metrics?

Reply:
We have android-only metrics (The Android SDK version)

Reply:
`collected-data`, `collected-metrcis`, `data-collection`?

Reply:
> 
> 
> We have android-only metrics (The Android SDK version)

Yes, but they are marked as optional for this reason. If there's any iOS specific metric, we could mark it as optional as well, and mention that it is iOS specific. What do you think?

> 
> 
> `collected-data`, `collected-metrcis`, `data-collection`?

`collected-metrics` sounds good.